### ISSUE TITLE
Cannot replay epoch info generation for epoch height 1055

### ISSUE THREAD

**Issue by Longarithm** [near/nearcore#11476]
Epoch height 1055 is around the last height with protocol version 48 used. Apparently what matters is that we started to use `ProtocolFeature::AliasValidatorSelectionAlgorithm` after that. Looks like we can't replay the old behaviour, attempt to do so results in the first validator index diverging.

It is not visible and has moderate urgency, because everyone seems to use already generated epoch infos, but if one tries to regenerate epoch infos from genesis, their chain is likely to diverge.

**Comment by wacban** [comment]
Is this something that needs to be solved for epoch sync?

**Comment by Longarithm** [comment]
Yeah, looks like [the doc](https://docs.google.com/document/d/14Itc9Hs7ewTRmcGANid9UCaYcJzaLzzM7FsvYYIFKDY) includes recomputation of epoch infos. cc @posvyatokum
